packages/web/config/price.ts

```diff
@@ -457,4 +457,17 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     spotPriceDestDenom: "uosmo",
     destCoinId: "osmosis",
   },
+  {
+    alternativeCoinId: "pool:stustars",
+    poolId: "810",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-75" }],
+      "stustars"
+    ),
+    spotPriceDestDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-0" }],
+      "ustars"
+    ),
+    destCoinId: "cosmos",
```

Comment:
"cosmos" is ATOM's CGid. If opting for a CoingGecko ID, then use STARS' ("stargaze"). If opting for a local Osmosis spot price, then use "pool:ustars" (this gets the price from the STARS/OSMO pool)

Reply:
fixed, thank you!